### PR TITLE
feat: Add builtin function of env

### PR DESCRIPTION
**Pull request by LeaYeh**
* if error occur in env function current process will be exit with error code

### CODE REVIEW

File: source/builtins/env.c [hunk 1]
```diff
@@ -6,9 +6,68 @@
...
+		i++;
+	}
+	free(str_array);
+}
+
+void	ft_exec_env(t_shell *shell, char **args)
+{
+	if (!exec_env(shell, args))
+	{
+		free_str_array(args);
```
Comment (itislu) [inline source/builtins/env.c:68]: If exec_env returns false, second call to free_str_array in line 72 will result in segfault in line 56.

Is there any reason args should be freed before the shell is cleaned?
- If not, delete line 68 and the args array will be freed after the shell was cleaned anyway.
- If yes, put an "else" in front of line 72.

File: source/builtins/env.c [hunk 2]
```diff
@@ -6,9 +6,68 @@
...
+{
+	int	i;
+
+	i = 0;
+	while (args[i])
+		i++;
+	if (i > 1)
+	{
+		ft_putstr_fd("env: ", 2);
+		ft_putstr_fd(args[1], 2);
```
Comment (itislu) [inline source/builtins/env.c:27]: Bash puts the arguments into apostrophes ( ‘ ’ ), like this: 
> env: ‘argument’: No such file or directory

Can we add this character [ ‘ ] at the very end of "env: " and at the very beginning of ": No such file or directory\n"?

File: source/builtins/env.c (hunk 1 above)
Comment (itislu) [reply source/builtins/env.c:68]: Resolved: ft_clean_shell exits the program.

**Review by itislu — CHANGES_REQUESTED** [review]
To review:
- [x] Formatting of env error message.

**Comment by LeaYeh** [comment]
@itislu solved it please check again.

**Review by itislu — APPROVED** [review]

**Pull request merged**